utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -178,19 +178,39 @@ def _cleanup_by_path(self, path):
 
 
 class OscapScan(object):
-    def __init__(self, tmp_dir=tempfile.gettempdir(), mnt_dir=os.getcwd(),
+    def __init__(self, tmp_dir=tempfile.gettempdir(), mnt_dir=None,
                  hours_old=2):
         self.tmp_dir = tmp_dir
         self.helper = OscapHelpers(tmp_dir)
         self.mnt_dir = mnt_dir
         self.hours_old = hours_old
 
+    def _get_mnt_dir(self):
+        '''
+        Get directory for mount container
+        '''
```

Comment:
Yes, good point. I will try to improve name, but I don't know how to comment it better. Comment message longer than method code isn't good way. Especially in so short non public method.

I will amend last commit with the changes.